packages/side-nav/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
We need to add a reference to this project in the root `tscnofig.json` file so that when we run `yarn ts` from the root it picks up and properly type checks this package.